crates/interledger-packet/src/packet.rs

```diff
@@ -643,6 +646,33 @@ impl From<Prepare> for BytesMut {
     }
 }
 
+#[cfg(test)]
+mod fuzzed {
+    use super::Packet;
+    use bytes::BytesMut;
+    use std::convert::TryFrom;
+
+    #[test]
+    fn fuzzed_0_non_utf8_error_code() {
```

Comment:
```suggestion
    fn fuzzed_0_non_ascii_error_code() {
```

I'll handle this in the follow-up.